### PR TITLE
Resolves #2450

### LINKED ISSUE #2450: WhatIf switch is not working
Hi Team,
During a simple test discovered the WhatIf switch (-WhatIf) was not working:

```
PS C:\Windows\system32> New-SmbShare -FullAccess everyone -Path C:\Shares\fff -Name fff -Description "DFS entry point 'f
ff'" -WhatIf

Name ScopeName Path          Description
---- --------- ----          -----------
fff  *         C:\Shares\fff DFS entry point 'fff'


PS C:\Windows\system32> New-SmbShare -FullAccess everyone -Path C:\Shares\fff -Name fff -Description "DFS entry point 'f
ff'"
New-SmbShare : The name has already been shared.
At line:1 char:1
+ New-SmbShare -FullAccess everyone -Path C:\Shares\fff -Name fff -Desc ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (MSFT_SMBShare:ROOT/Microsoft/Windows/SMB/MSFT_SMBShare) [New-SmbShare], C
   imException
    + FullyQualifiedErrorId : Windows System Error 2118,New-SmbShare

PS C:\Windows\system32> $psversiontable

Name                           Value
----                           -----
PSVersion                      5.1.14393.4350
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.4350
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
Just to let you know!!

Regards,

kees

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 69d4fd86-7df0-9f6f-acf9-ee51ea28aa0d
* Version Independent ID: bd1a3a35-9e2a-25c3-7387-d72fd31f4231
* Content: [New-SmbShare (SmbShare)](https://docs.microsoft.com/en-us/powershell/module/smbshare/new-smbshare?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/smbshare/New-SmbShare.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/smbshare/New-SmbShare.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Add whatif note
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2450

### CODE REVIEW

File: docset/winserver2012-ps/smbshare/New-SmbShare.md [hunk 1]
```diff
@@ -210,14 +210,14 @@ The acceptable values for this parameter are:
  -- AccessBased: SMB will not the display the files and folders for a share to a user unless that user has rights to access the files and folders.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/New-SmbShare.md:210]:
```suggestion
 -- AccessBased: SMB will not display the files and folders for a share to a user unless that user has rights to access the files and folders.
```

File: docset/winserver2012r2-ps/smbshare/New-SmbShare.md [hunk 2]
```diff
@@ -229,14 +229,14 @@ The acceptable values for this parameter are:
  -- AccessBased: SMB will not the display the files and folders for a share to a user unless that user has rights to access the files and folders.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/New-SmbShare.md:229]:
```suggestion
 -- AccessBased: SMB will not display the files and folders for a share to a user unless that user has rights to access the files and folders.
```

File: docset/winserver2016-ps/smbshare/New-SmbShare.md [hunk 3]
```diff
@@ -244,14 +244,14 @@ The acceptable values for this parameter are:
 SMB does not the display the files and folders for a share to a user unless that user has rights to access the files and folders.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/smbshare/New-SmbShare.md:244]:
```suggestion
SMB does not display the files and folders for a share to a user unless that user has rights to access the files and folders.
```

File: docset/winserver2019-ps/smbshare/New-SmbShare.md [hunk 4]
```diff
@@ -244,14 +244,14 @@ The acceptable values for this parameter are:
 SMB does not the display the files and folders for a share to a user unless that user has rights to access the files and folders.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/smbshare/New-SmbShare.md:244]:
```suggestion
SMB does not display the files and folders for a share to a user unless that user has rights to access the files and folders.
```

File: docset/winserver2022-ps/smbshare/New-SmbShare.md [hunk 5]
```diff
@@ -244,14 +244,14 @@ The acceptable values for this parameter are:
 SMB does not the display the files and folders for a share to a user unless that user has rights to access the files and folders.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/New-SmbShare.md:244]:
```suggestion
SMB does not display the files and folders for a share to a user unless that user has rights to access the files and folders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @velkovb, please commit my additional suggestions. Many thanks.

File: docset/winserver2022-ps/smbshare/New-SmbShare.md [hunk 6]
```diff
@@ -412,6 +412,9 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
```
Comment (get-itips) [inline docset/winserver2022-ps/smbshare/New-SmbShare.md:415]: Are we allowing Note tags in cmdlet articles now? maybe I don't know the last news about this...

File: docset/winserver2012-ps/smbshare/New-SmbShare.md [hunk 7]
```diff
@@ -376,10 +376,13 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
+> The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/New-SmbShare.md:380]:
```suggestion
Note: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2012r2-ps/smbshare/New-SmbShare.md [hunk 8]
```diff
@@ -396,6 +396,9 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
+> The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/New-SmbShare.md:400]:
```suggestion
Note: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2022-ps/smbshare/New-SmbShare.md (hunk 6 above)
Comment (JohanFreelancer9) [reply docset/winserver2022-ps/smbshare/New-SmbShare.md:415]: @get-itips - Thank you, Andres, missed it. No, Note tags are not allowed.

Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/New-SmbShare.md:415]:
```suggestion
```

File: docset/winserver2022-ps/smbshare/New-SmbShare.md [hunk 9]
```diff
@@ -412,6 +412,9 @@ Accept wildcard characters: False
...
+> The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/New-SmbShare.md:416]:
```suggestion
Note: The WhatIf switch does not work with this cmdlet.
```

File: docset/winserver2016-ps/smbshare/New-SmbShare.md [hunk 10]
```diff
@@ -412,6 +412,9 @@ Accept wildcard characters: False
 Shows what would happen if the cmdlet runs.
 The cmdlet is not run.
 
+> [!NOTE]
+> The WhatIf switch does not work with this cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/smbshare/New-SmbShare.md:416]:
```suggestion
Note: The WhatIf switch does not work with this cmdlet.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @get-itips and @JohanFreelancer9.

The last word was to avoid using the formatted notes.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@velkovb - Please commit my suggested changes to correct the note format. Many thanks.

**Comment by velkovb** [comment]
We always have this discussion. Last PR I had was with the formatted notes. 
https://github.com/MicrosoftDocs/windows-powershell-docs/pull/2206

**Comment by dariomws** [comment]
These are the final words:

"In cmdlet’s articles the [!NOTE] (also [!IMPORTANT] & [!CAUTION]) should not be used because it can’t be rendered later by the Get-Help cmdlet in the console window."

I'll fwd you an email @velkovb.

**Comment by JohanFreelancer9** [comment]
In #2206 you have inserted the note text under the **NOTE** heading. @velkovb @get-itips @dariomws - What do you think that is the best option here?

**Comment by dariomws** [comment]
@JohanFreelancer9 let's avoid using formatted notes. I'll send you the same email. Thanks!

**Comment by velkovb** [comment]
My bad then, I guess I was too sleepy. Do we put it in Notes heading or just under the parameter?

**Comment by dariomws** [comment]
Just under the parameter is ok.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**